.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
```

Comment:
- [ OPTIONAL ] Please make sure that your commits messages are in present tense and they describe an action
![image](https://user-images.githubusercontent.com/74744787/159786592-3d1fc29b-233d-4182-9701-49a39ab7f3c6.png)